stdlib/public/core/StringBridge.swift

```diff
@@ -658,7 +658,7 @@ internal func _SwiftCreateBridgedString_DoNotCall(
 // This allows us to subclass an Objective-C class and use the fast Swift
 // memory allocator.
 @objc @_swift_native_objc_runtime_base(__SwiftNativeNSStringBase)
-class __SwiftNativeNSString {
+@_spi(Foundation) public class __SwiftNativeNSString {
```

Comment:
Trying to minimize the ABI exposure here. I could expose __StringStorage and __SharedStringStorage, but since this is their superclass, I would need to expose it anyway, and this is sufficient for hanging @objc implementations off of that operate in terms of the new initializer.